src/app/db/db_Jobs.py

```
@@ -0,0 +1,147 @@
+            """,
+            (job_type, "pending"),
+        )
+        rows = self.db.fetch_query_safe(
+            """
+            SELECT id, job_type, status, started_at, completed_at, result_file, created_at, updated_at
+            FROM jobs
+            WHERE id = LAST_INSERT_ID()
+            """
+        )
+        if not rows:
+            raise RuntimeError("Failed to create job")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)